packages/spacecat-shared-data-access/src/index.d.ts

```diff
@@ -421,6 +421,12 @@ export interface Configuration {
    */
   getHandler: (type) => object;
 
+  /**
+   * Retrieves the slack configuration.
```

Comment:
doc doesn't match what the method does

Reply:
changed it to "slack roles configuration"